manifests/install.pp

```diff
@@ -103,14 +110,13 @@
 
   if $jira::db == 'mysql' and $jira::mysql_connector_manage {
     if $jira::staging_or_deploy == 'staging' {
-      class { '::jira::mysql_connector':
+      class { 'jira::mysql_connector':
```

Comment:
While you no longer need colon prepends in Puppet 4, I imagine they should persist as long as Puppet 3 is supported.